README.md

```diff
@@ -1,33 +1,19 @@
-# Token-Curated Registry
+# Futarchy Curated Registry
 
-[ ![Codeship Status for skmgoldin/tcr](https://app.codeship.com/projects/b140cce0-ac77-0135-0738-52e8b96e2dec/status?branch=master)](https://app.codeship.com/projects/257003)
+## Run solc-compile (debugging only)
 
-A string-keyed [token-curated registry (TCR)](https://medium.com/@ilovebagels/token-curated-registries-1-0-61a232f8dac7).
+This scripts allows you to make changes to a contract and recompile without the added overhead of `truffle compile`.
 
-## Initialize
-The only environmental dependency you need is Node. Presently we can guarantee this all works with Node 8.
-```
-npm install
-npm run compile
-```
+To compile a single contract and it's imports:
```

Comment:
I think I understand that compiling single contracts is something you do repeatedly.  But the flow of this seems a little weird because one of the first steps under "compile a single contract" is to "Compile all contracts".  Maybe could say "To compile a single contract repeatedly"

Reply:
@jpkcambridge I reworded it a bit